examples/tutorials/rigid_object_primitive_tutorial.py

```diff
@@ -0,0 +1,198 @@
+# [setup]
+import math
+import os
+from pathlib import Path
+
+import cv2
+import magnum as mn
+import numpy as np
+
+import examples.tutorials.tutorial_functions as tutFuncs
+import habitat_sim
+import habitat_sim.utils.common as ut
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+data_path = os.path.join(dir_path, "../../data")
+output_path = os.path.join(dir_path, "rigid_object_primitive_tutorial_output/")
+
+
+def make_video_cv2(
+    observations, camera_res=[540, 720], prefix="", open_vid=True, multi_obs=False
+):
+    videodims = (camera_res[1], camera_res[0])
+    fourcc = cv2.VideoWriter_fourcc("m", "p", "4", "v")
+    video = cv2.VideoWriter(output_path + prefix + ".mp4", fourcc, 60, videodims)
+    thumb_size = (int(videodims[0] / 5), int(videodims[1] / 5))
+    outline_frame = np.ones((thumb_size[1] + 2, thumb_size[0] + 2, 3), np.uint8) * 150
+
+    if multi_obs:
+        for ob in observations:
+            embed_image_data_list = [
+                (ob["rgba_camera_1stperson"], False),
+                (ob["depth_camera_1stperson"], True),
+            ]
+
+            res_image = tutFuncs.build_multi_obs_image(
+                ob["rgba_camera_3rdperson"],
+                embed_image_data_list,
+                thumb_size,
+                outline_frame,
+            )
+            # write the desired image to video
+            video.write(res_image)
+    else:
+        for ob in observations:
+            res_image = ob["rgba_camera_1stperson"][..., 0:3][..., ::-1]
+
+            # write the desired image to video
+            video.write(res_image)
+    video.release()
+    if open_vid:
+        os.system("open " + output_path + prefix + ".mp4")
```

Comment:
This should be os.startfile to be more platform agnostic.

Reply:
os.startfile seems to only be supported/available on windows. 

https://docs.python.org/3/library/os.html